config/sites/admissions.uiowa.edu/views.view.areas_of_study.yml

```diff
@@ -1097,7 +1097,7 @@ display:
           plugin_id: text
           empty: false
           content:
-            value: "<hr class=\"element--spacer-separator\" />\r\n<h2>Tests</h2>\r\n<table>\r\n  <thead>\r\n    <tr>\r\n      <th scope=\"col\">Page Name</th>\r\n      <th scope=\"col\">Operations</th>\r\n    </tr>\r\n  </thead>\r\n  <tbody>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/461\" title=\"Cambridge International Exams (CIE)\">Cambridge International Exams (CIE)</a></td>\r\n      <td><a href=\"/node/461/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/451\" title=\"College Level Examination Program (CLEP)\">College Level Examination Program (CLEP)</a></td>\r\n      <td><a href=\"/node/451/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/456\" title=\"International Baccalaureate Program (IB)\">International Baccalaureate Program (IB)</a></td>\r\n      <td><a href=\"/node/456/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/446\">Advanced Placement Program (AP)</a></td>\r\n      <td><a href=\"/node/446/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/4211\">Global Assessment Certificate Program (GAC)</a></td>\r\n      <td><a href=\"/node/4211/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n  </tbody>\r\n</table>"
+            value: "<hr class=\"element--spacer-separator\" />\r\n<h2>Tests</h2>\r\n<table>\r\n  <thead>\r\n    <tr>\r\n      <th scope=\"col\">Page Name</th>\r\n      <th scope=\"col\">Operations</th>\r\n    </tr>\r\n  </thead>\r\n  <tbody>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/461\" title=\"General Certificate of Education (GCE)\">General Certificate of Education (GCE)</a></td>\r\n      <td><a href=\"/node/461/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/451\" title=\"College Level Examination Program (CLEP)\">College Level Examination Program (CLEP)</a></td>\r\n      <td><a href=\"/node/451/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/456\" title=\"International Baccalaureate Program (IB)\">International Baccalaureate Program (IB)</a></td>\r\n      <td><a href=\"/node/456/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/446\">Advanced Placement Program (AP)</a></td>\r\n      <td><a href=\"/node/446/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/4211\">Global Assessment Certificate Program (GAC)</a></td>\r\n      <td><a href=\"/node/4211/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n  </tbody>\r\n</table>"
```

Comment:
Not totally clear on what all is supposed to be in here but this looks wrong: `<td><a href=\"/node/456/edit\" media_library=\"Media Library\">edit</a></td>\r\n`

Reply:
Yes, that is pretty bizarre to see!